tests/get_maintainer_wrapper_test.sh

```diff
@@ -50,147 +51,91 @@ Maintainers already in \"To:\" field of update_patch_test.patch"
 
 FAKE_KERNEL="tests/.tmp"
 
-function setupGetMaintainers
+function oneTimeSetUp
 {
   # This creates tests/.tmp which should mock a kernel tree root. A .git
   # dir is also created inside tests/.tmp so that get_maintainer.pl thinks
   # it is a git repo. This is done in order to avoid some warnings that
   # get_maintainer.pl prints when no .git is found.
   mk_fake_kernel_root "$FAKE_KERNEL"
-  mkdir -p "$FAKE_KERNEL/.git"
+  cp -rf tests/samples/fake-git/ "$FAKE_KERNEL"/.git
```

Comment:
How about introducing a new helper function for creating a fake git repository? Something similar to mk_fake_kernel_root. Maybe you can add this in a separate commit introduced before this one.

Reply:
The `.git` fake folder created in this commit is very specific to the warning it's trying to stop. It has one single commit with one single file under control: `fs/nothing`. So I don't think it's general enough to become it's own function. On the other hand, most test modules that call `mk_fake_kernel_root` also create an empty `.git` folder (the only exception is the `checkpath_wrapper_test` module). My proposal: this particular case remains as is in my commit and we include the creation of a `.git` folder as a part of the `mk_fake_kernel_root` function, in another PR. Which makes sense, after all, the kernel wouldn't be the kernel weren't it for git. What do you think, @rodrigosiqueira?

Reply:
> Which makes sense, after all, the kernel wouldn't be the kernel weren't it for git.

Actually, some people can download the kernel tarball and compile/install without any git requirement :)

Instead of statically create this fake git folder, can we at least generate it dynamically in the oneTimeSetup function?

Reply:
Okay! In this new version we dynamically generate a `.git` folder inside the `oneTimeSetup` function.